Readme.md

```diff
@@ -1,22 +1,35 @@
-<p>The Items collection contains items of two types: </p><p>1)ViewModel objects - contain data for creating DevExpress.Xpf.Docking.LayoutPanels </p><p>2)DocumentViewModel objects - contain data for creating DevExpress.Xpf.Docking.DocumentPanels </p><p>There are two templates declared in XAML that render input objects as either DevExpress.Xpf.Docking.LayoutPanels or DevExpress.Xpf.Docking.DocumentPanels. </p><br />
-<p>The custom DockItemTemplateSelector class overrides the SelectTemplate method to choose a template that corresponds to a specific item in the DevExpress.Xpf.Docking.DockLayoutManager.ItemsSource. This template selector is assigned in XAML to the DevExpress.Xpf.Docking.DockLayoutManager.ItemTemplateSelector property. </p><br />
-<p>In this example, the DevExpress.Xpf.Docking.DockLayoutManager contains two groups explicitly created in XAML: a DevExpress.Xpf.Docking.LayoutGroup named panelHost and a DevExpress.Xpf.Docking.DocumentGroup named documentHost. Items from the Items collection are automatically placed in one of these groups based on their IMVVMDockingProperties.TargetName property values. </p>
-
-<br/>
-
-
+<!-- default badges list -->
+[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E4867)
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+
+# WPF Dock Layout Manager - Generate a Dock UI with the MVVM Design Pattern
+
+The following creates [LayoutPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.LayoutPanel) and [DocumentPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DocumentPanel) objects from a data source according to the MVVM design pattern and implements template selection.
+
+<img src="https://user-images.githubusercontent.com/12169834/175355745-d50cc84d-c36e-4f4b-9db5-7b0dbd211b8a.png" width=600px/>
+
+In this example, the **MainViewModel** class contains the **Items** collection that contains objects that are displayed as layout items in the [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager). An instance of the **MainViewModel** class is assigned to the Window's **DataContext**. The [DockLayoutManager.ItemsSource](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager.ItemsSource) property is bound to the **Items** property in the **MainViewModel**. In this case, the [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) gets the source of objects to  visualize it as layout items.

```

Comment:
`In this case, the [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) gets the source of objects to  visualize it as layout items.`

Does "it" refer to "objects"? If so, do you mean "them" instead? For example:

`In this case, the [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) gets the source of objects to  visualize them as layout items.`

Reply:
```suggestion
In this example, the **MainViewModel** class contains the **Items** collection that contains objects that are displayed as layout items in the [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager). An instance of the **MainViewModel** class is assigned to the Window's **DataContext**. The [DockLayoutManager.ItemsSource](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager.ItemsSource) property is bound to the **Items** property in the **MainViewModel**. In this case, the [DockLayoutManager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) gets the source of objects to visualize them as layout items.

```